charts/datadog/README.md

```diff
@@ -483,7 +483,7 @@ helm install --name <RELEASE_NAME> \
 | datadog.logs.containerCollectAll | bool | `false` | Enable this to allow log collection for all containers |
 | datadog.logs.containerCollectUsingFiles | bool | `true` | Collect logs from files in /var/log/pods instead of using container runtime API |
 | datadog.logs.enabled | bool | `false` | Enables this to activate Datadog Agent log collection |
-| datadog.networkMonitoring.enabled | bool | `false` | Enable network performance monitoring |
+| datadog.networkMonitoring.enabled | bool | `nil` | Enable network performance monitoring |
```

Comment:
Let's make explcitit that systemProbe.enabled is used if `nil`

Reply:
sorry but a bool have only two value: `true` and `false`.
Can we get ride of these three states?

Reply:
I explained the reason here: https://github.com/DataDog/helm-charts/pull/164#discussion_r577845106
Without the `nil` value we can't preserve backwards compatibility and customers would inadvertently disable NPM monitoring.